github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+|                    | area-test                |                  |                 |
+|                    | area-testability         |                  |                 |
+|                    | area-transport           |                  |                 |
+|                    | area-needs-triage        |                  |                 |
+|                    | area-try-service-fabric  |                  |                 |
+|                    | varea-windows-server     |                  |                 |
+|                    |                          |                  |                 |
+
+Project - NA for now
+--------------------------------------------------------------------------
+
+| ***xxxx*** | ***xxx*** |
+|------------|-----------|
+| xxx        | xxx       |
+
+
+> **No labels for this area at this time.**
+
+Milestones
+----------------------------------------------------------------
+
+| ***Release (Iteration path)***                                                            |
+|-------------------------------------------------------------------------------------------|
+| 6.3, 6.4, etc. (define specifically once we get a build number at end of release)         
+                                                                                            
+ 6.3 CU1, 6.4 CU1, etc. (define specifically once we a get build number at end of release)  
+                                                                                            
+ > Example 6.3.xxx.xx                                                                       
+ >                                                                                          
+ > All work items update automatically once the milestone is updated. (= minimal effort)    
+                                                                                            
+ There is not a vNext or backlog release.                                                   
+                                                                                            
+ > No Milestone = backlog                                                                   |
+
+Happy Path By issue
+===================
+
+Path below based on current State/Phase Labels
+
+****Question Happy Path****
+
+*Open -&gt; triaged -&gt; needs-more-info (optional) -&gt; in-progress -&gt; Closed. *
+
+****Bug or Documentation Happy Path****
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; need-more-info (optional) -&gt; understood (optional) -&gt; in -progress -&gt; Closed. *
+
+****Enhancement Happy Path****
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; design -&gt; in -progress -&gt; review -&gt; Closed. *
+
+> ****Questions, Defects and Doc issues****
+>
+> ***'Optional' Phases will apply to issues that are immediately understood allowing them to bypass many of the phases, moving the bug right to 'in-progress'.***
```

Comment:
this makes it sound like you're expecting developers to specifically follow these flows.  Why not just replace all of this with 
> Describe your issues using these four variables: area, status, type, and an optional priority flag

I think it's harmful/confusing to attempt to predict how the labels will be used, and write them up as **the** process.